pkg/controller/servicebinding/reconciler_test.go

```diff
@@ -509,6 +532,10 @@ func TestEmptyApplication(t *testing.T) {
 	requireConditionPresentAndTrue(t, CollectionReady, sbrOutput.Status.Conditions)
 	requireConditionPresentAndFalse(t, InjectionReady, sbrOutput.Status.Conditions)
 	requireConditionPresentAndTrue(t, BindingReady, sbrOutput.Status.Conditions)
+
+	bindingSecret, err := fakeDynClient.Resource(secretsGVR).Get(sbrOutput.Status.Secret, metav1.GetOptions{})
+	require.NoError(t, err)
+	require.Equal(t, sbrOutput.Status.Secret, bindingSecret.GetName())
```

Comment:
`bindingSecret.GetName()` will always be equal to `sbrOutput.Status.Secret` if no error is raised, so this test is irrelevant.